routes/sapiV1CapiV2.js

```diff
@@ -129,4 +130,129 @@ router.get('/test', async (req, res, next) => {
 	});
 });
 
+function prepAnnotationsGroup( groupName, annoPairs, groupDetails, searchResponse ){
+  const group = {
+    name : groupName,
+    byCount : {
+      topAnnotations : [],
+      annotationsBubblingUnder : [],
+    },
+  };
+
+  group.byCount.topAnnotations = annoPairs
+  .filter( pair => { return pair[1] > 1; }) // just those with count > 1
```

Comment:
Would there be a case where `pair[1]` is a String? might be a good thing to `parseInt()` just in case.

Reply:
Should never be a string. Is always constructed within the code based on the length of an array